samples/operate-first/Makefile

```diff
@@ -0,0 +1,20 @@
+ROOT_DIR := ../..
+include $(ROOT_DIR)/Makefile.env
+export RELEASE_NAME=m4d
+export NAMESPACE=m4d-system
+export M4D_CHART_PATH=$(ROOT_DIR)/charts/m4d
+export CRD_CHART_PATH=$(ROOT_DIR)/charts/m4d-crd
+
+.PHONY: helm-template
+helm-template: $(TOOLBIN)/yq $(TOOLBIN)/helm
+	$(TOOLBIN)/helm template $(RELEASE_NAME) -n $(NAMESPACE) $(M4D_CHART_PATH) | $(TOOLBIN)/yq e 'select(.kind=="Namespace" or .kind=="MutatingWebhookConfiguration" or .kind=="ValidatingWebhookConfiguration" or .kind=="ClusterRole" or .kind=="ClusterRoleBinding")' - > m4d.yaml
+	$(TOOLBIN)/helm template $(RELEASE_NAME) -n $(NAMESPACE) $(CRD_CHART_PATH) > m4d-crd.yaml
```

Comment:
using the same release name will work because we don't use it in the crd templates, but if we ever change that then it would be better to use a unique release name like `m4d-crd`